recipes/amplici/build.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [[ `uname` == "Darwin" ]]; then
+    export CONFIG_ARGS="-DCMAKE_FIND_FRAMEWORK=NEVER -DCMAKE_FIND_APPBUNDLE=NEVER"
+else
+    export CONFIG_ARGS=""
+fi
+
+cd src
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)